typeDefs.js

```diff
@@ -278,9 +279,9 @@ const typeDefs = gql`
     updateUserByToken(user: UserInput!): User
 
     "Product Mutation"
-    addProduct(product: productInput!): Product
-    deleteProduct(id: ID!): Product
-    updateProduct(id: ID!, product: productInput!): Product
+    addProduct(product: ProductInput!): ProductResult
+    deleteProduct(id: ID!): ProductResult
+    updateProduct(id: ID!, product: ProductInput!): ProductResult
 
     "Comment Mutation"
     addComment(comment: commentInput!): Comment
```

Comment:
It should be `CommentResult` because this operation can throw an error. Also, please, define `productId` as the first argument and data for the creation of comment and the second. It will make mutation much more readable

Reply:
Done